inc/saiport.h

```diff
@@ -695,6 +695,28 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_FDB_LEARNING_LIMIT_VIOLATION,
 
+    /** Bind/Unbind an ingress acl or acl-group on a port [sai_object_id_t].
+     *      Enable (or bind) ingress ACL filtering by assigning the ACL table 
+     *      object id or ACL group object id as attribute value. 
+     *      Disable (or unbind) ingress filtering by assigning 
+     *      SAI_NULL_OBJECT_ID in the attribute value.
+     *      Default Value: SAI_NULL_OBJECT_ID
+     */
+    SAI_PORT_ATTR_INGRESS_ACL_ID,
```

Comment:
The OID list will contain both the ACL group OIDs and ACL table OIDs. The priorities of the ACL tables - outside of the group as well as part of the group should be resolved. considering that binding point. This may seem complex but there is no prioirty for aCL group, so ACL tables is all we need. And ofcourse resolving non-conflicting actions across different OID objects.